staging/src/k8s.io/mount-utils/resizefs_linux.go

```diff
@@ -84,3 +86,16 @@ func (resizefs *ResizeFs) xfsResize(deviceMountPath string) (bool, error) {
 	resizeError := fmt.Errorf("resize of device %s failed: %v. xfs_growfs output: %s", deviceMountPath, err, string(output))
 	return false, resizeError
 }
+
+func (resizefs *ResizeFs) btrfsResize(deviceMountPath string) (bool, error) {
+	args := []string{"filesystem", "resize", "max", deviceMountPath}
+	output, err := resizefs.mounter.Exec.Command("btrfs", args...).CombinedOutput()
```

Comment:
```suggestion
	output, err := resizefs.exec.Command("btrfs", args...).CombinedOutput()
```
we have changed from mounter to exec here.

Reply:
thanks for the direction, I didn't realise more than just the file paths got refactored earlier today